src/Dfc.CourseDirectory.Web/Controllers/QualificationsController.cs

```diff
@@ -44,51 +44,6 @@ public IActionResult Index()
             _session.SetString("Option", "Qualifications");
             return View();
         }
-        [Authorize]
```

Comment:
I've blammoed this already in PTCD-470 https://github.com/SkillsFundingAgency/dfc-coursedirectory/pull/1684/files#diff-c1e2ecbdac03e77504ef5fb8461a85c7L48